examples/gradle-example/src/test/java/com/autonomic/tmc/example/auth/GradleAuthExampleTest.java

```diff
@@ -0,0 +1,55 @@
+package com.autonomic.tmc.example.auth;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.eq;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class GradleAuthExampleTest {
+
+    @Autowired
+    private GradleAuthExample example;
+
+    @Captor
+    private ArgumentCaptor<Exception> errorMessages;
+
+    @Spy
+    LoggerStub loggerStub;
+
+    @Test
+    void givenBadCredentials_whenAuthenticating_thenVerifyErrorMessageContainsSdkVersion() {
+        example.LOGGER = loggerStub;
+        example.run("hi");
+        Mockito.verify(example.LOGGER).log(
+            eq(Level.SEVERE),
+            eq("Generic message, Something went wrong: "),
+            errorMessages.capture());
+        String actualError = errorMessages.getAllValues().get(0).getMessage();
+
+        assertTrue(actualError.contains("tmc-auth"));
```

Comment:
This fails for both myself and @ahararwala when running locally.

Reply:
We had a discussion and agree this is an expected scenario for now.